src/igvfd/schemas/changelogs/auxiliary_set.md

```diff
@@ -2,6 +2,7 @@
 
 ### Minor changes since schema version 11
 
+* Add `MORF barcode sequencing`.
```

Comment:
This sentence is used for adding new properties, I think the one we want here is:
> Extend `file_set_type` enum list to include `...`.

Reply:
Addressed.